book-2-platform-api/chapters/API_MODELS_CONTROLLERS.md

```diff
@@ -96,70 +158,121 @@ namespace CoffeeShop.Controllers
         [HttpGet]
         public async Task<IActionResult> Get()
         {
-            using (IDbConnection conn = Connection)
+            using (SqlConnection conn = Connection)
             {
-                string sql = "SELECT * FROM Coffee";
+                conn.Open();
+                using (SqlCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = $"SELECT Id, Title, BeanType FROM Coffee";
+                    SqlDataReader reader = cmd.ExecuteReader();
+                    List<Coffee> coffees = new List<Coffee>();
 
-                var coffees = await conn.QueryAsync<Exercise>(sql);
-                return Ok(coffees);
-            }
+                    if (reader.Read())
```

Comment:
this will only ever get one coffee row.  did you mean `while(reader.Read())`?

Reply:
Good catch, thanks